spec/datadog/profiling/stack_recorder_spec.rb

```diff
@@ -344,8 +347,8 @@ def sample_types_from(decoded_profile)
       let(:labels) { { 'label_a' => 'value_a', 'label_b' => 'value_b', 'state' => 'unknown' }.to_a }
 
       let(:a_string) { 'a beautiful string' }
-      let(:an_array) { [1..10] }
-      let(:a_hash) { { 'a' => 1, 'b' => '2', 'c' => true } }
+      let(:an_array) { (1..100).to_a }
```

Comment:
Minor: I believe doing `(1..100).to_a.compact` will guarantee that the underlying array has a capacity of exactly 100.